### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by JohnRDOrazio**
Translations update from [Weblate](https://translate.johnromanodorazio.com) for [Confessit Web/Public Locales](https://translate.johnromanodorazio.com/projects/confessit-web/public-locales/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -1,427 +1,427 @@
...
-    "if_you_find": "If you find this app useful, please consider <strong>sharing it</strong> with your friends and family. Tell people about it on Facebook, Reddit, Twitter, at your church, or in your Bible study group to help spread the word!",
-    "privacy": "Privacy",
-    "information_you_enter": "Information you enter into this app is only stored on your device. It is not sent over the internet. We are able to do this using a technology provided by your web browser called <a>local stroage</a>. We do not run Google Analytics or any other data collection mechanism on this site. Data you enter will be saved on your device until you hit <kbd>Clear</kbd> even if you close the window or refresh the page.",
-    "open_source": "Open Source",
-    "confessit_is_open_source": "ConfessIt is open source. We develop the app on <i></i> <github>GitHub</github> and we collaborate in the <osc>Open Source Catholic</osc> community on Slack.",
```
Comment (kas-catholic) [inline public/locales/en/translation.json:60]: @JohnRDOrazio This `<i></i>` was correct. It represents the GitHub fontawesome icon in our React i18n string - see the rendered version near the bottom of https://confessit.app/about

Comment (JohnRDOrazio) [reply public/locales/en/translation.json:60]: oh my, I would have expected to see something like `<i className="fab fa-github"></i>`. I'll add a note in weblate for all translators explaining what this is. Just out of curiosity, how does that then become the github icon?

Comment (JohnRDOrazio) [reply public/locales/en/translation.json:60]: should a github icon appear here too? https://translate.johnromanodorazio.com/translate/confessit-web/public-locales/en/?checksum=cf7c69f51e27cb84

Comment (JohnRDOrazio) [reply public/locales/en/translation.json:60]: I see now where it's being transformed:

https://github.com/kas-catholic/confessit-web/blob/c987b52e39adc99fc26b93bcbb26ae7572825999/src/About.js#L204-L214

**Review by kas-catholic — APPROVED** [review]

**Pull request merged**